core/workflow-api/src/main/java/org/apache/syncope/core/workflow/api/UserWorkflowAdapter.java

```diff
@@ -60,6 +61,10 @@
     WorkflowResult<Pair<String, Boolean>> create(
             UserTO userTO, boolean disablePwdPolicyCheck, final Boolean enabled, boolean storePassword);
 
+
+    WorkflowResult<String> doRequestCertify(final User user);
```

Comment:
`do*` methods are defined by abstract implementations for inner execution: please rename this method as `requestCertify`